test/Tizen.NUI.Samples/Tizen.NUI.Samples/Tizen.NUI.Samples.csproj

```diff
@@ -25,7 +25,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Tizen.NET" Version="8.0.0.15045" />
+    <PackageReference Include="Tizen.NET" Version="8.0.0.15134" />
```

Comment:
Could you refer project folder instead of nuget.
If you use nuget, it cannot be built because code is not synced.